class.jetpack-cli.php

```diff
@@ -291,22 +302,95 @@ public function reset( $args, $assoc_args ) {
 				WP_CLI::line( __( "Resetting default modules...\n", "jetpack" ) );
 				usleep( 500000 ); // Take a breath
 				$default_modules = Jetpack::get_default_modules();
-				Jetpack::update_active_modules( $default_modules );
+				if ( ! $is_dry_run ) {
+					Jetpack::update_active_modules( $default_modules );
+				}
 				WP_CLI::success( __( 'Modules reset to default.', 'jetpack' ) );
 
 				// Jumpstart option is special
-				Jetpack_Options::update_option( 'jumpstart', 'new_connection' );
+				if ( ! $is_dry_run ) {
+					Jetpack_Options::update_option( 'jumpstart', 'new_connection' );
+				}
 				WP_CLI::success( __( 'jumpstart option reset', 'jetpack' ) );
 				break;
 			case 'modules':
-				$default_modules = Jetpack::get_default_modules();
-				Jetpack::update_active_modules( $default_modules );
+				if ( ! $is_dry_run ) {
+					$default_modules = Jetpack::get_default_modules();
+					Jetpack::update_active_modules( $default_modules );
+				} else {
+					WP_CLI::success( __( 'This is a `Dry Run` and module activation will be reset', 'jetpack' ) );
+				}
+
 				WP_CLI::success( __( 'Modules reset to default.', 'jetpack' ) );
 				break;
 			case 'prompt':
-				WP_CLI::error( __( 'Please specify if you would like to reset your options, or modules', 'jetpack' ) );
+				WP_CLI::error( __( 'Please specify if you would like to reset your options, modules or sync-checksum', 'jetpack' ) );
 				break;
+			case 'sync-checksum':
+				global $wpdb;
+				$option = 'jetpack_callables_sync_checksum';
+
+				if ( $is_dry_run ) {
+					WP_CLI::success( __( 'This is a `Dry Run` and no options will be deleted', 'jetpack' ) );
+				}
+
+				if ( is_multisite() && function_exists( 'get_sites' ) ) {
+					$sites  = get_sites( array( 'number' => 1000 ) );
+					$count_fixes  = 0;
+					foreach ( $sites as $site ) {
+						switch_to_blog( $site->blog_id );
+						$count = self::count_option( $option );
+						if ( $count > 1 ) {
+							if ( $is_dry_run ) {
+								WP_CLI::line( sprintf( __( 'DRY RUN: Deleted %s %s options from %s', 'jetpack' ), $count, $option, "{$site->domain}{$site->path}" ) );
```

Comment:
I think one "dry run" message in the beginning and one at the end of the command should be enough. This could help keep this simpler as well. What do you think?

Reply:
nice! good call :)